Sources/GPX/OAGPXDatabase.h

```diff
@@ -15,8 +15,7 @@
 
 @interface OAGPX : NSObject
 
-@property (nonatomic) NSString *gpxFileName;
-@property (nonatomic) NSString *gpxFilepath;
+@property (nonatomic) NSString *file;
```

Comment:
Why did you change this? Java's file corresponds to Obj C's NSString *filePath. Please revert this